msg/CartesianTrajectoryPoint.msg

```diff
@@ -0,0 +1,14 @@
+geometry_msgs/Pose pose
+geometry_msgs/Twist velocity
+geometry_msgs/Accel acceleration
+
+geometry_msgs/Pose pose_upper_tolerance
+geometry_msgs/Pose pose_lower_tolerance
+
+geometry_msgs/Twist velocity_upper_tolerance
+geometry_msgs/Twist velocity_lower_tolerance
+
+geometry_msgs/Accel acceleration_upper_tolerance
+geometry_msgs/Accel acceleration_lower_tolerance
```

Comment:
For all three of these groups I don't see how the lower-upper combination is useful really.
What is the "lower velocity tolerance" supposed to be? A lower limit?
Please feel free to disagree, but we are talking about derivatives of a 6 dimensional space here. Even the 6d pose space cannot be bound very well with only two bounds, not to talk about the derivatives. So you impose a one dimensional manifold with the current message which seems not very useful to me.

I would propose to reduce these fields to two numbers each for position tolerance (in m, m/s, m/s^2) and orientation tolerance (rad, rad/s, rad/s^2). This leaves us with a tolerance/uncertainty measure without imposing an artificial 1d interval.
If you want to represent your bounds then, you can still use three CartesianTrajectories, but personally I would go for Motion primitives instead.

Reply:
Indeed. The main reason for not using a pose or transform directly to describe the cartesian part was to address Dave's comment:
> This won't support tolerances, which is important for Descartes
I basically tried to mirror the Descartes limits as a first draft. Indeed, it is a lower limit. 

I slightly disagree with using a single limit as for planar plans for instance, one could allow higher tolerances on the Z axis, while having tight ones on X and Y. As the planners I'm working with doesn't really support tolerances, I don't have a strong position on this one.

Reply:
@mlautman has worked with MoveIt! <> Descartes integration a lot recently, please weight in on the need for these

Reply:
I agree we don't need lower and upper bounds for velocity and acceleration

I do think we need the ability to specify different tolerances between XYZ, RPY

Reply:
Descartes uses pose tolerances but it doesn't have any velocity or acceleration upper or lower limits.

Reply:
I think constraints are a little overkill. `CartesianTrajectoryPoint` is just a point in time and should not have a concept of tolerances. The MotionPlanRequest already has Constraints that can be used for tolerances